lib/remote/apilistener.cpp

```diff
@@ -311,6 +311,7 @@ void ApiListener::Stop(bool runtimeDeleted)
 
 ApiListener::Ptr ApiListener::GetInstance()
 {
+
```

Comment:
Please

* [ ] undo this line
* [ ] let me review again

Reply:
Seriously, this is not how we treat contributors. Instead of writing this you could've amended and merged it.